hoomd/md/pair.py

```diff
@@ -764,6 +764,12 @@ def set_from_file(self, a, b, filename):
 class Morse(_Pair):
     R""" Morse pair potential.
 
+    Args:
+        nlist (:py:mod:`hoomd.md.nlist`): Neighbor list
```

Comment:
```suggestion
        nlist (hoomd.md.nlist._Nlist): Neighbor list
```
And similarly for all other potential classes.